app/src/main/java/org/fossasia/openevent/app/event/detail/EventDetailPresenter.java

```diff
@@ -46,12 +48,27 @@ public void attach(IEventDetailView eventDetailView, long initialEventId) {
         this.initialEventId = initialEventId;
     }
 
+    private void loadAll(boolean refresh) {
```

Comment:
I think presenter should not even reset when orientation change happens. Many MVP libraries have implemented this. I think we should focus on presenter view implementation itself. Including a param will be a workaround to this problem, not a solution, not to mention that then in future, we'll have to implement it in every presenter. This will break DRY (Don't repeat yourself). That same if else everywhere. I'm studying this https://github.com/quangctkm9207/mvp-android-arch-component to apply architecture components in MVP, so it'll solve both our problems